pkgs/applications/misc/bikeshed/default.nix

```diff
@@ -0,0 +1,71 @@
+{ lib
+, buildPythonApplication
+, fetchPypi
+# build inputs
+, aiofiles
+, aiohttp
+, attrs
+, certifi
+, cssselect
+, html5lib
+, isodate
+, json-home-client
+, lxml
+, pillow
+, pygments
+, requests
+, result
+, setuptools
+, tenacity
+, widlparser
+}:
+
+buildPythonApplication rec {
+  pname = "bikeshed";
+  version = "3.3.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-eIr6AxBgX/cArO9qEYG5jnSiaDIddYiDbAYNqjcnrNc=";
+  };
+
+  patches = [
+     # Relax requirements from "==" to ">="
```

Comment:
if all the `==` are being replaced to `>=` then this can be done
```
  postPatch = ''
    substituteInPlace requirements.txt \
      --replace "==" ">="
  '';
```

Reply:
good point! I'll try that